source/linux/Foundational_Components/Graphics/SGX/Build_Guide.rst

```diff
@@ -0,0 +1,238 @@
+###############
+SGX Build Guide
+###############
+
+The graphics stack on SGX has been frozen. The last major change occured in
+release 09.01. This will capture the build process and some of the caviots of
+this legacy driver.
+
+This driver has been reworked as part of the kernel 6.1 support effort to mimic
+the build and structure of rogue. The stack consists of 3 main components with
+varying classifications:
+
+   - The kernel module (KM) (Open Source, but out-of-tree)
+   - The GLES 2.0 implementation (UM) (Closed Source, binary release)
+   - The mesa shim (Open Source, but out-of-tree)
+
+All 3 of these components are required for GLES support.
+
+*****************
+SGX Kernel Module
+*****************
+
+The kernel module is responsible for:
+
+   - Memory and power management
+   - Monitoring device state
+   - Orchestrating hardware recoveries
+   - Creating an interface for the userspace server
+
+The source for the component is available at:
+
+   https://git.ti.com/cgit/graphics/omap5-sgx-ddk-linux
+
+Building the SGX Kernel Module
+==============================
+
+The latest branch contains the most bugfixes and has been adapted for the widest
+general compatibility in modern applications. This is
+``1.17.4948957/mesa/k6.1``, and despite being tagged for kernel 6.1 it is also
+interoperable with kernel 6.6.
+
+The :file:`INSTALL` file gives a brief overview of how to build and install the
+module. This gives you the common variables, but it does leave out some of the
+more important ones for the build. The following variables are key and *must*
+match with the values used in the UM release:
+
+   - ``BUILD`` -- Options: ``release`` or ``debug``
+   - ``PVR_BUILD_DIR`` -- Options: ``<platform>_linux``, for example ``ti572x_linux``
+   - ``WINDOW_SYSTEM`` -- Options: ``lws-generic``, ``xorg``, ``wayland``, ...
+
+These variables must be set for the build. To check for updated build variables
+please see the ``EXTRA_OEMAKE`` variable in the `ti-sgx-ddk-km recipe in
+meta-ti`_. This will always contain the latest required variables to compile the
+driver.
+
+.. _ti-sgx-ddk-km recipe in meta-ti: https://git.ti.com/cgit/arago-project/meta-ti/tree/meta-ti-bsp/recipes-bsp/powervr-drivers/ti-sgx-ddk-km_1.17.4948957.bb
```

Comment:
@StaticRocket Won't this link eventually be broken once the PV gets updated from 1.17.4948957?

Reply:
SGX PV will never change. It's forever stuck in 1.17 as that's the last official release.